src/coreclr/build-test.cmd

```diff
@@ -613,10 +613,13 @@ set __TotalPrecompiled=0
 set __FailedToPrecompile=0
 set __FailedAssemblies=
 for %%F in ("%CORE_ROOT%\System.*.dll";"%CORE_ROOT%\Microsoft.*.dll") do (
-    if not "%%~nxF"=="Microsoft.CodeAnalysis.VisualBasic.dll" if not "%%~nxF"=="Microsoft.CodeAnalysis.CSharp.dll" if not "%%~nxF"=="Microsoft.CodeAnalysis.dll" (
+    if not "%%~nxF"=="Microsoft.CodeAnalysis.VisualBasic.dll" (
+    if not "%%~nxF"=="Microsoft.CodeAnalysis.CSharp.dll" (
+    if not "%%~nxF"=="Microsoft.CodeAnalysis.dll" (
+    if not "%%~nxF"=="System.Runtime.WindowsRuntime.dll" (
         call :PrecompileAssembly "%%F" %%~nxF __TotalPrecompiled __FailedToPrecompile __FailedAssemblies
```

Comment:
Non-blocking nit: looks much better than before but it might be even easier to move this logic to the top of PrecompileAssembly as a series of independent bail-out clauses like

<pre>
if "%~nx1"=="Microsoft.CodeAnalysis.VisualBasic.dll" goto EXIT
</pre>

or some such. I certainly don't insist :-).

Reply:
I actually prefer to filter it before calling the PrecompileAssembly :-)